src/store/modules/Health/EventLogStore.js

```diff
@@ -0,0 +1,68 @@
+import api from '../../api';
+
+const EventLogStore = {
+  namespaced: true,
+  state: {
+    eventLogData: null
+  },
+  getters: {
+    eventLogData(state) {
+      return state.eventLogData;
+    }
+  },
+  mutations: {
+    setEventLogData(state, eventLogData) {
+      state.eventLogData = eventLogData;
+    }
+  },
+  actions: {
+    getEventLogData({ commit }) {
+      api
+        .get('/xyz/openbmc_project/logging/enumerate')
+        .then(response => {
+          const eventLog = response.data.data;
+          const entryNumber = /[1-9]/;
+          const eventLogEntries = [];
+          const severityToPriorityMap = {
+            Emergency: 'High',
+            Alert: 'High',
+            Critical: 'High',
+            Error: 'High',
+            Warning: 'Medium',
+            Notice: 'Low',
+            Debug: 'Low',
+            Informational: 'Low'
+          };
+          for (let key in eventLog) {
+            if (
+              key.includes('entry') &&
+              key.match(entryNumber) &&
+              !key.includes('callout')
+            ) {
+              const eventKey = eventLog[key];
+              const eventSeverity = eventKey.Severity.split('.').pop();
+              const eventPriority = severityToPriorityMap[eventSeverity];
+              eventLogEntries.push(
+                Object.assign(
+                  {
+                    logId: '#' + eventKey.Id,
+                    severity: eventPriority,
```

Comment:
I think we calling the Alert, Critical, Error, Warning field "Severity"  and the high, medium, low field "Priority", so should this be priority: eventPriority